### PR TITLE
Human only commandos (nuke ops, deathsquad, ERT)

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
**You still get to roll for these jobs if you're a lizard, you just get set to human species**

ERT/Deathsquad/Nuke ops can only have human members.

We don't let lizards be heads of staff, so it shatters my immersion when I see them in roles that outrank heads of staff/are the best of the best.

In the interest of fairness to lizard players (and so I don't have to figure out something stupid and complicated with configs) this is not a blacklist, it just force sets their species as human if their ghost is pulled.

EDIT: I just realized this is totally a pre-emptive bug fix if plasmamen ever get added to stop them igniting if they spawn as nuke ops please merge this is critical and not at all driven by lizard hate.

### CODE REVIEW

**Comment by CosmicScientist** [comment]
Do they look alright after they're converted to human?

**Comment by KorPhaeron** [comment]
@CosmicScientist 

Better than they did as lizards. Most of them have helmets and gas masks on anyway.

**Comment by Cheridan** [comment]
Don't see any reason why there can't be lizard nuke ops.

**Comment by xxalpha** [comment]
:-1:

File: code/modules/admin/verbs/one_click_antag.dm [hunk 1]
```diff
@@ -348,6 +348,9 @@
 			missionobj.completed = 1
 			Commando.mind.objectives += missionobj
 
+			//Humans only
+			Commando.set_species("human")
```
Comment (phil235) [inline code/modules/admin/verbs/one_click_antag.dm]: if you update their species you must update their dna. Put this right before Commando.dna.update_dna_identity() on line 335

**Comment by SweaterKittens** [comment]
I agree with Deathsquad/ERT, but I'm with Cheridan on them being allowed to be Nuke Ops. It actually makes a lot of sense thematically that opposing corporations might pit lizards against a company that seems to harbor a lot of racism towards them.

**Comment by Incoming5643** [comment]
Kor where did the lizard touch you?

**Comment by KorPhaeron** [comment]
@Incoming5643 

I just spawned an ERT and every one of them followed orders and finished their objective save for the lizard they can not be trusted

**Comment by Xhuis** [comment]
thumbsup

**Comment by Cheridan** [comment]
Don't you see, though? That guy wouldn't have followed orders either way, at least if he's a lizard you know beforehand that he's awful.

**Comment by KorPhaeron** [comment]
(F) [Common] Triggers-The-Kor hisses, "I can't wait to be a piece of ssshit and rub my lizard handsss all over everything."

I want Cheridan removed from reviewing my PR he is clearly biased and mad with power

**Comment by Incoming5643** [comment]
It is legitimately humorous to me that people are taking the "they're only shitty players because they're playing lizards!" argument seriously.

If someone wants to be a dick they're going to do it, species doesn't matter. Especially when they're almost always in a full body spacesuit that hides all features. No, lizards are just a conduit through which dicks flow through, probably almost exclusively because they can give themselves a meme name. You'll notice that this is entirely an administration problem and not something that really needs a code solution.

**Comment by Steelpoint** [comment]
I agree that Death Squad and Emergency Response Team members should be Human only, I also throw in my support as the person responsible for the creation of the ERT for the server (for whatever that's worth).

However I disagree on forcing Nuke Ops to be Human only, while I would love that (maybe make it a leader preference and make being a leader a actual antag option) but what was stated above is true in that I would imagine the Syndicate would be more than happy to employ disenfranchised Lizards into their ranks who have a bone to grind against Nanotrasen.

**Comment by Zilenan91** [comment]
I don't see the problem with having lizard ERT's/Squids/Reps, it creates some pretty funny RP situations. One of the ones I remember is the captain starting to totally flip his shit when I walk into the brig during a rev round with no implant as a lizard. It ended up with me implanted and the whole crew asking me why a lizard was their boss.

Plasmamen shouldn't be any of these though, since they'll just die instantly. That's probably the only real problem there is with this.

**Comment by CosmicScientist** [comment]
Just a reminder (thanks Kor for pointing out the plasmamen bursting into flames issue with nuke ops), please add the human race only (or a whitelist or something that future proofs these roles that can be amended) to other antags that would need it (e.g. wizard) otherwise the same thing will happen there.

**Comment by Steelpoint** [comment]
But again, we already restrict Lizards from very important and high level jobs in Nanotrasen, that's even enshired into the lore, then why do we allow them into a high level rescue team or a elite top secret special forces squad.

If NT does not trust a Lizard with being in a command position onboard one of its premium space station's, why would they trust them with the most advance weapons and equipment in the know universe and to use them in the interests of NT.

**Comment by Zilenan91** [comment]
Why does NT trust lizards to make literal bombs, robotic mech suits with enough arms and armor to level a small country, completely unknown guns and backpacks that may or may not be able to create black holes, and also trust them to make super plagues that can kill entire stations. ERT's/Squids are child's play in terms of IC complexity compared to that.

**Comment by Steelpoint** [comment]
Except the Death Squad can crush all of that easily.

The DS are really powerful, their armour is the most powerful in the entire game by a long margin and their weapon will destroy any mech in a few hits, not to mention they are a two hit crit on any human target. They are effectively immune to most attacks and are near stun immune.

Not to mention if the DS bring their heavy duty combat mechs to bear then its all over.

The technology the crew of SS13 can make is really impressive and cutting edge, but its just that. New technology not yet refined. The gear the DS use has been around for a while and has been refined for usage.

**Comment by Supermichael777** [comment]
H:8ball:. stop hate-coding away lizards this is literally admin-spawn. :-1:

**Comment by Zilenan91** [comment]
I wasn't saying that NT couldn't deal with anything the station throws at them, what I was saying is that why does NT, IC, trust dirty lizards to literally jump hundreds of years forward in scientific fields in a very short timespan

Think about it. Think about the amount of math and shit IC that goes into making just Xrays, about how they have to literally make it so that pure energy goes through walls and people. Then think about things like sentience potions. A potion that literally jumps the intelligence of animals millions upon millions of years forward and makes them smart enough to intelligently communicate and have the same mental capacity as a human does. Both of these can be done by lizards.

**Comment by Bawhoppen** [comment]
Remove lizards from actual job roles then, thats the only true solution

(no but for real, people who play as lizards do that intentionally to show how they want to act like shitlers and dicks, but of course then anti-lizard people use the RP/lore angle to try and go against those kind of players and playstyles while they can't really do that openly)

**Comment by Supermichael777** [comment]
what to people have against the lizard wizard?

**Comment by Steelpoint** [comment]
Your assuming that its only Lizards that are doing research and development.

At this point all we know lore wise of Lizards is that most Humans distrust them. NT only hired them to cut costs on hiring Human employee's and shore up gaps in the work force, thus implying Lizards are underpaid in comparison.

I wish we had stats on the amount of humans versus non-humans in the average crew.

But I digress, thematicly it makes sense, and being correct from a lore point of view, that non-humans are kept out of high level positions.

You say it adds more roleplay to have Lizards in these positions, however I counter in saying it's just as valid (from a PR pov) that Lizards be kept out of these positions as it is for them to be in them.

--

Overall this change is a mostly minor one, ERT are a rare spawn, DS are even rarer and the Nuke Ops generally can't be seen underneath all that armour they wear.

Its a minor change I support, so I'm going to end my side in this debate with a simple

:+1:

**Comment by Zilenan91** [comment]
I see your point, but I still disagree with having lizards being restricted from any of these. If this is merged, they should still be in Nuke Ops because there's no real logical reason for them not to be.

**Comment by Steelpoint** [comment]
Correction, as I said above I support Human only ERT/DS but I disagree with Human only Ops.

I would support Human only Ops only if it was a toggle for the Op leader in a pref menu.

**Comment by Bawhoppen** [comment]
isn't the point that the majority of humans are racist against lizards, no matter if they're syndie or NT?

**Comment by Zilenan91** [comment]
There's never been any lore that's ever said that, so I'm going with the syndicate not being racist towards lizards. They probably don't give two shits who they are if they get the job done.

**Comment by Bawhoppen** [comment]
@Zilenan91 https://tgstation13.org/wiki/Lizardperson

**Comment by Steelpoint** [comment]
Also before you ask that piece of lore was made not long after the introduction of Lizards and was put to a community vote, with the majority voting in favour of keeping and using it.

**Comment by Zilenan91** [comment]
No I mean syndicate attitude towards lizards, not NT.

**Comment by Bawhoppen** [comment]
> All alien species, including Lizardmen, have always been treated as second class citizens in Human society.

**Comment by Supermichael777** [comment]
using lore as an excuse for hate-code.

**Comment by Zilenan91** [comment]
Except the syndicate is a company that clearly deals in a lot of crime. We don't know if they actually have, y'know, people living in service of them like NT does, or even if they have any space stations at all. They might just hire people to do the things they want for them.

**Comment by Bawhoppen** [comment]
You argue for the lore but you don't even seem to know it.

**Comment by Supermichael777** [comment]
would you kill this man for 1000 space dollars?
Sure
okay heres a way to get a gun.

**Comment by Dorsisdwarf** [comment]
would you be kitted out in some of the most advanced gear known outside of the Supercorp NT, then put onto a stealth shuttle with 4 commandos you've never met before, then handed access to millitary hardware and told to assault a heavily-defended research outpost?
Sure
okay here's a way to our top-secret forwards listening post and the codes for our stolen nuclear device

**Comment by Incoming5643** [comment]
To be clear I get the notion that deathsquad and ERT lizards don't make a lot of lore sense, I just think the reasoning being used is hilariously silly and mostly an excuse to grudgecode.

Maybe this should be a thing for deathsquads and ERTs, but should be tied to that config var that limits tlizards from command positions.

---

On the plasmaman front, we actually already have a var that serves this purpose already: dangerous_existence. So that's all kind of moot.

**Comment by Jacquerel** [comment]
Is this hardcoded or is it a server config

**Comment by bgobandit** [comment]
What we know lore-wise of lizards is Nanotrasen is actually the LEAST racist corporation against lizards.

**Comment by Iamgoofball** [comment]
REMEMBER WHEN WE SAID THAT LIZARDS WERE GONNA BE ADMIN SPAWN ONLY

**Comment by Saegrimr** [comment]
This is effectively a buff for nuke ops/ert/squaddies because now the AI can't valid their ass on arrival.

**Comment by Lumipharon** [comment]
Ops should be able to be lizards.

**Comment by Y0SH1M4S73R** [comment]
I was a lizard op one time and I carried my squad, even going so far as to retrieve a spaced nuke. I just want to be a lizard to be a lizard, not to be a useless piece of shit that gets nothing done. DNM

**Comment by Bawhoppen** [comment]
dont be a lizard then if you want to get work done, the whole point is you're a worthless piece of asimov valid shit

**Comment by Supermichael777** [comment]
saegamer actually brings up a good point. but i thought ais couldn't get their valids on until it was confirmed non human.

**Comment by Dorsisdwarf** [comment]
nuke op winrates didnt go down when lizards were added, hence this wouldn't be likely to increase them

**Comment by Lumipharon** [comment]
gameplay aside, there is no fluff reason to exclude lizards from nuke ops - they're not NT, they don't give a fuck about species - nuke ops are combat crazy warboys.

**Comment by PKPenguin321** [comment]
>Don't see any reason why there can't be lizard nuke ops.

asimov

**Comment by Kromgar** [comment]
RACE WAR NOW GAS THE LIGGERS

**Comment by Lumipharon** [comment]
Pk what? What does asimov have to do with it? If you CHOOSE to be a lizard, you're willingly giving up your human protections.

Lizards are restricted on the NT side because of fluff reasons, not gameplay.

**Comment by Xhuis** [comment]
antilizardb

**Comment by PKPenguin321** [comment]
>Pk what? What does asimov have to do with it? If you CHOOSE to be a lizard, you're willingly giving up your human protections.

if you like playing lizard AND like playing nuke ops, you get cucked by asimov
this allows you to play both roles without disadvantage
if anything it's just making the game more fair to lizard players

**Comment by Steelpoint** [comment]
The only way a Asimov AI/Cyborg will know your a Lizard Operative is if they manage to strip off your clothing (can you even remove Op armour anymore?) or you speak.

Otherwise your armour does a good job of hiding your species identity.

**Comment by PKPenguin321** [comment]
...so? this change will enable lizards to speak without outing themselves, it's still a "buff" for lizard players either way

that, and ops dont always wear their hardsuits all the time (for example, stealth ops)

**Comment by Lumipharon** [comment]
>buff for lizard playing is not being a lizard

Also this would apply to literally all antags - or hell, an situation what so ever that involves danger, harm or humans.

**Comment by PKPenguin321** [comment]
yeah pretty much
it's more vital for nuke ops specifically though, since it's such a team-oriented mode and the failure of one op could mean the failure of all of them
let them be lizards when they're just fucking around as assistants, but let them be humans when it really matters what their species is

**Comment by Wyzack** [comment]
No lizards in the ERT and Deathsquad makes a lot of sense, merge he.
Ligger ops i am pretty ambivalent on, people have brought up some good points as to why they should exist

**Comment by Lumipharon** [comment]
What about ligger wizard? Any you get (other then bleb and ling), your non human status can potentially be a hinderance. However you chose to be a lizard, and you chose to have the chance to be X antag.

Also nuke ops not being lizards ever has the (very niche) meta of being able to trust lizards inherently, even if stealth ops are around.

**Comment by Steelpoint** [comment]
What if someone does not want to be a non-human for certain antags?

The game offers no way to distinguish someone's choice, its either all or nothing.

For Ops (and any other antag) there should be a option to tick that allows you to default to a Human appearance when your selected for a off-station antag, such as Ops or Wizard.

**Comment by Incoming5643** [comment]
Wizard doesn't need anything like that, magic mirror is already essentially that.

**Comment by duncathan** [comment]
@Lumipharon I completely agree that if someone chooses to play as a ligger, they should have to face any consequences that come from it. The thing is, with ops, the other ops **didn't** choose to play with a ligger. So it does put the team at ever so slightly more of a disadvantage, in a completely unavoidable way for the others.

**Comment by Bawhoppen** [comment]
In my opinion in ops (and wiz, ert, deathsquid, centcomm official, and pretty much anything else,)  you should be given a random body anyways

**Comment by tuypo1** [comment]
@SweaterKittens auctualy under current lore nanotrasen has nothing against lizards they only keep them from head roles so they don't lose contracts from the racist government

its my favourite part of the current lore

**Comment by Steelpoint** [comment]
@tuypo1 ehh it really depends on how you look at it.

**Comment by tuypo1** [comment]
I suppose it is in some ways a little vague

as for people worried about stealth ops on both sides of the argument fuck stealth ops stealth ops are fucking cancer

on the matter of the pr itself im not in favour of if. the idea of letting people be in your spec ops teams when you would not not let them into lower groups is perfectly reasonable getting the job done goes above petty hatred im sure if I went looking I could find plenty of examples of female special forces in male only military's throughout history

**Comment by Dorsisdwarf** [comment]
"You lizards are scum, unworthy of authority"

"Here, be part of this super-elite rescue force that outranks everyone on-station/part of this kill-team sent to judge and terminate threats/a high admiral"

**Comment by Jacquerel** [comment]
Is this linked to the human authority config or not?  
Makes sense for it to be active when that config is active, does not when it is not.

**Comment by Dorsisdwarf** [comment]
Jacquerel makes a good point. Our lore is not the only lore

**Comment by KorPhaeron** [comment]
Yeah but the other lore is shit

**Comment by Bawhoppen** [comment]
Again I say, for these types of roles it should give you a random body and a random name, which will also make it that way you can't tell like 'X robust player is with us, I'll stay on this ops team since we prolly won't fluke'

**Comment by tuypo1** [comment]
@Bawhoppen 
I agree I am against static names and appearances in general outside of silicons and entertainment rolls but its doubly important for nuke ops

**Comment by KorPhaeron** [comment]
Unfortunately our random body generation tends to create women with purple beards (or it did last I checked), so I don't really have an interest in doing that.

Beyond the scope of this PR anyway.

**Comment by Bawhoppen** [comment]
Whenever I rolled random bodies, I always get edge-tier dunkers

**Comment by Miauw62** [comment]
i dont think it still makes bearded women but it still gives them male names.

also it doesnt make lizards

**Comment by SpookyDoc** [comment]
Keep Liz Ops. The Syndicate makes deals with fucking space wizards and shit. Hell, one of the composite parts of the Syndicate is made up entirely of rogue silicones. I don't think The Syndicate cares who you are as long as you're willing to undermine Nanotrasen.

**Comment by Dorsisdwarf** [comment]
@KorPhaeron nerd confix

**Comment by Zilenan91** [comment]
Can we lock this by now?

**Comment by KorPhaeron** [comment]
Nah I'm gonna remove the syndie restriction and fix it

**Comment by KorPhaeron** [comment]
You have not seen the last of this

**Comment by Xhuis** [comment]
goodnight sweet prince ;_;7

**Pull request closed without merging**